### PR TITLE
[TM-1513] add attemps for delayed jobs

### PR DESCRIPTION
**Pull request by egrojMonroy**
After a 502 on getting job status, try 3 times .

### CODE REVIEW

File: src/generated/apiFetcher.ts [hunk 1]
```diff
@@ -145,6 +145,12 @@ async function loadJob(signal: AbortSignal | undefined, delayedJobId: string): P
 
     const url = resolveV3Url(`/jobs/v3/delayedJobs/${delayedJobId}`);
     response = await fetch(url, { signal, headers });
+
+    if (response.status === 502 && retries > 0) {
```
Comment (roguenet) [inline src/generated/apiFetcher.ts:149]: Please add a comment explaining why we're doing this; if I were a developer coming across this code in 6 months, I'd be pretty confused about why this is here.

File: src/generated/apiFetcher.ts [hunk 2]
```diff
@@ -136,7 +136,7 @@ type JobResult = {
   };
 };
 
-async function loadJob(signal: AbortSignal | undefined, delayedJobId: string): Promise<JobResult> {
+async function loadJob(signal: AbortSignal | undefined, delayedJobId: string, retries: number = 3): Promise<JobResult> {
```
Comment (roguenet) [inline src/generated/apiFetcher.ts:139]: 🎨 There's no reason to specify the type when a default value is provided; TS compiler will do just fine understanding it.

File: src/generated/apiFetcher.ts (hunk 1 above)
Comment (egrojMonroy) [reply src/generated/apiFetcher.ts:153]: Great, a comment is added.

File: src/generated/apiFetcher.ts (hunk 2 above)
Comment (egrojMonroy) [reply src/generated/apiFetcher.ts]: removed

**Review by roguenet — APPROVED** [review]

**Pull request merged**